lib/converter/rule.go

```diff
@@ -84,8 +84,8 @@ func ruleAPIToBackend(ar api.Rule) model.Rule {
 		NotICMPType: notICMPType,
 
 		SrcTag:      ar.Source.Tag,
-		SrcNet:      ar.Source.Net,
-		SrcNets:     ar.Source.Nets,
+		SrcNet:      normalizeIPNet(ar.Source.Net),
```

Comment:
I guess it doesn't matter too much, but the converter code is now obsolete - so not really necessary to update this - guess it doesn't hurt though.

Reply:
will still be used if we do a 2.7, right?

Reply:
not in master, nope